src/components/AddUserMenu.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="popup-menu">
+    <form @submit.prevent="handleAddUser">
+      <div class="form-group" v-bind:class="{ invalid: submitted && missingEmail }">
+        <input
+          type="text"
+          placeholder="Email"
+          v-model="email"
+          :class="{ 'is-invalid': submitted && missingEmail }"
+        />
+        <div v-show="submitted && missingEmail" class="error">Email is required</div>
+      </div>
+      <div class="form-group">
+        <div class="check-input">
+          <input
+            type="checkbox"
+            id="admin"
+            value="admin"
+            v-model="admin"
+          >
+          <label for="admin">Admin</label><br>
+        </div>
+      </div>
+      <div class="form-group">
+        <button type="submit" :disabled="status.addUserRequest">
+          Add User
+        </button>
+      </div>
+    </form>
+    <p v-if="alert.type === 'alert-danger'" class="error">
+      {{ alert.message }}
```

Comment:
We need to not use the alert module for these messages since it propagates throughout the other error messages. I'll make an issue for this.